ui/components/app/signature-request-original/signature-request-original.component.js

```diff
@@ -327,6 +354,14 @@ export default class SignatureRequestOriginal extends Component {
           </div>
         ) : null}
         {this.renderFooter()}
+        {messagesCount > 1 ? (
+          <a
```

Comment:
We should use a `<button>` component for accessibility instead of the `<a>`

Reply:
@darkwing updated.